lap3dkernel/c++SIMD/lap3dkernel.cpp

```diff
@@ -25,6 +25,7 @@
 
 // choose double prec throughout
 typedef double Real;
+typedef long Integer;
```

Comment:
This line should be removed. Integer is not used anywhere. It was used in pow function, but I removed the pow function in latest commit. Should find this before create the PR, sorry. 😐